### ISSUE TITLE
Library should not us the Ti namespace

### ISSUE THREAD

**Issue by rblalock** [guilhermechapiewski/titanium-cache#3]
First, awesome library and great idea.  Second, you should use another namespace other than the Ti namespace.  Anything with the Ti namespace is subject to change and could break anything you try to add to it.  Don't look at it as a true javascript object.

I recommend doing this as a CommonJS module so a user can require() it and call it whatever he wants (while protecting the module's scope).

Great job on this.

**Comment by ordinz** [comment]
+1.  This looks awesome, good job!

**Comment by x-ji** [comment]
Yeah not to mention currently Ti.include is actually already deprecated. CommonJS module is a must. Great library though. Probably someone else has to pick up the work here.

**Comment by 779nhan** [comment]
0868242510 code
